enhancements/windows-containers/windows-machine-config-operator-upgrades.md

```diff
@@ -60,7 +60,7 @@ will then be able to configure VMs that will be created in place of the
 terminated ones.
 
 In order to allow for this to occur with minimal service disruption, it is
-recommended that users have at least two Windows nodes within the cluster.
+recommended that users have at least three Windows nodes within the cluster.
```

Comment:
What's the reason for this change?

Reply:
With the new upgrade process, we are deleting and reconfiguring nodes one at a time.
Having 3 Windows nodes ensures high availability and avoids any downtime for the workloads.

For eg. when we have only 2 nodes during upgrade, there will be a downtime for Windows workloads, when the workload is scaling up from 0 to 1 on newly upgraded node.

Reply:
@sdodson should we explicitly call that out?

Reply:
No, this is probably fine. I was thinking about saying we should say something like "N+1 capacity" but that gets complicated quickly since it's not quite that simple.

I was mostly wondering if this was driven by new critical omponent being added to the Windows stack for OpenShift which must be HA.

Reply:
>I was mostly wondering if this was driven by new critical omponent being added to the Windows stack for OpenShift which must be HA.

Nothing like that as of now. We just observed downtime on workloads with just two nodes during upgrades and wanted to address that.